terraform/locals.tf

```diff
@@ -82,7 +82,7 @@ locals {
     },
     {
       name  = "SPELLING_CORRECTOR_BUCKET_NAME"
-      value = "trade-tariff-search-configuration-${var.environment}-${local.account_id}"
```

Comment:
I was thinking about this recently. I guess we are going to have to get familiar with the account ids to know which environment we're in